1-js/02-first-steps/16-function-expressions/article.md

```diff
@@ -1,84 +1,84 @@
 # Function expressions
```

Comment:
I'm aware of the fact that there is no suitable equivalent in Persian for `Function expressions`, but **titles** should always be translated because they'll be in RTL and they won't look good this way.
For example you can use `اعلان تابع Expression`.